app/routes/__index/dns-records/instructions.tsx

```diff
@@ -1,21 +1,153 @@
-import { Heading, Text, Link } from '@chakra-ui/react';
+import { InfoOutlineIcon } from '@chakra-ui/icons';
+import {
+  Heading,
+  Text,
+  Container,
+  Grid,
+  VStack,
+  UnorderedList,
+  ListItem,
+  OrderedList,
+  Link as ChakraLink,
+  Accordion,
+} from '@chakra-ui/react';
+import { Link } from '@remix-run/react';
+import FaqAccordion from '~/components/instructions/faq-accordion';
 
 export default function CertificateInstructionsRoute() {
   return (
-    <div>
-      <Heading as="h1" size="3xl">
-        DNS-Record Instructions Page
+    <Container maxW="container.md" py="8">
+      <Heading as="h1" size="xl" mb="4">
+        Dns Record Instructions
       </Heading>
-      <Text fontSize="2xl">
-        For information about this page, click{' '}
-        <Link
-          href="https://github.com/Seneca-CDOT/starchart/issues/443"
-          isExternal={true}
-          color="blue.500"
-        >
-          here
-        </Link>
-      </Text>
-    </div>
+      <Grid templateColumns="repeat(1, 1fr)" gap="4">
+        <VStack alignItems="flex-start">
+          <Heading as="h3" size="md">
+            What is a DNS Record?
+          </Heading>
+          <Text>
+            A DNS record is a type of data stored in a domain name system (DNS) server that maps a
+            domain name to an IP address or other information about the domain. DNS records enable
+            users to access websites using domain names instead of IP addresses, making it easier to
+            remember and navigate the internet.
+          </Text>
+        </VStack>
+        <VStack alignItems="flex-start">
+          <Heading as="h3" size="md">
+            Dns Record Types
+          </Heading>
+          <UnorderedList listStylePosition="inside" listStyleType="none">
+            <ListItem mb="3">
+              <Text fontWeight="medium">A Record</Text>
+              <Text>Purpose: Maps a domain name to an IPv4 address.</Text>
+              <Text>
+                Example: Your project's domain name is "my-project.user.mystudentproject.ca" and its
+                IPv4 address is "192.168.0.1". Create an A record to associate these two.
+              </Text>
+            </ListItem>
+            <ListItem mb="3">
+              <Text fontWeight="medium">AAAA Record</Text>
+              <Text>Purpose: Maps a domain name to an IPv6 address.</Text>
+              <Text>
+                Example: Your project's domain name is "my-ipv6-project.user.mystudentproject.ca"
+                and its IPv6 address is "2001:0db8:85a3:0000:0000:8a2e:0370:7334". Create an AAAA
+                record to associate these two.
+              </Text>
+            </ListItem>
+            <ListItem mb="3">
+              <Text fontWeight="medium">CNAME Record</Text>
+              <Text>Purpose: Maps a domain name to another domain name.</Text>
+              <Text>
+                Example: Your project's domain name is "my-project.user.mystudentproject.ca", and
```

Comment:
Add a note to this one explaining that you can't include other common parts of a URL (e.g., `https://` or a trailing `/`). It must ONLY be the domain, since this is a common mistake.